modules/oe_content_sub_entity/modules/oe_content_sub_entity_document_reference/tests/src/Functional/DocumentReferenceBundlesTest.php

```diff
@@ -0,0 +1,234 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_content_sub_entity_document_reference\Functional;
+
+use Drupal\Core\Entity\Entity\EntityFormDisplay;
+use Drupal\Core\Entity\Entity\EntityViewDisplay;
+use Drupal\Core\Field\FieldStorageDefinitionInterface;
+use Drupal\file\Entity\File;
+use Drupal\file\FileInterface;
+use Drupal\media\Entity\Media;
+use Drupal\media\MediaInterface;
+use Drupal\node\Entity\NodeType;
+use Drupal\Tests\BrowserTestBase;
+use Drupal\Tests\field\Traits\EntityReferenceTestTrait;
+use Drupal\Tests\TestFileCreationTrait;
+
+/**
+ * Tests the document reference bundles defined in this module.
+ */
+class DocumentReferenceBundlesTest extends BrowserTestBase {
+
+  use EntityReferenceTestTrait;
+  use TestFileCreationTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    'node',
+    'oe_content_sub_entity_document_reference',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $defaultTheme = 'classy';
```

Comment:
Note: Using 'classy' so that we can use field classes and entity classes as selectors.